apps/src/netsim/netsimConstants.js

```diff
@@ -75,3 +76,30 @@ exports.NetSimTabType = {
   ROUTER: 'router',
   DNS: 'dns'
 };
+
+/**
+ * Field types that can go into the packet header.
+ * Map to class-names that can be applied to related input fields.
+ * @enum {string}
+ */
+exports.PacketHeaderType = {
```

Comment:
The fact that these are being added to a constants class smells like poor encapsulation. I'd rather see a Packet class which knows how to encode/decode itself and in which this enum also lives. No need to do now, but worth considering whether this would simplify your code and may be worth doing eventually.